Makefile

```
@@ -40,6 +40,18 @@ release:
 	GOOS=linux GOARCH=arm64 go build -ldflags="-s -w -X github.com/docker/model-cli/desktop.Version=$(VERSION)" -o dist/linux-arm64/$(PLUGIN_NAME) .
 	@echo "Release build complete: $(PLUGIN_NAME) version '$(VERSION)'"
 
+ce-release:
+	@if [ -z "$(VERSION)" ]; then \
+		echo "Error: VERSION parameter is required. Use: make release VERSION=x.y.z"; \
+		exit 1; \
+	fi
+	@if [ "$(uname -s)" != "Linux" ]; then \
+		echo "Warning: This release target is designed for Linux"; \
+	fi
+	@echo "Building local release version '$(VERSION)'..."
+	GOOS=linux go build -ldflags="-s -w -X github.com/docker/model-cli/desktop.Version=$(VERSION)" -o dist/$(PLUGIN_NAME) .
```

```suggestion
	GOOS=linux GOARCH=amd64 go build -ldflags="-s -w -X github.com/docker/model-cli/desktop.Version=$(VERSION)" -o dist/linux-amd64/$(PLUGIN_NAME) .
	GOOS=linux GOARCH=arm64 go build -ldflags="-s -w -X github.com/docker/model-cli/desktop.Version=$(VERSION)" -o dist/linux-arm64/$(PLUGIN_NAME) .
```

Do you want to build for both arm64 and amd64?

---

Not for this one, this is built in the context of the target environment for `.deb`/`.rpm` package, so we want the `GOARCH` to match the expected target.